src/components/pages/Photos/cameraNavbar/CameraNavbar.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './CameraNavbar.css';
+
+export default function Navbar() {
+  return (
+    <ul className="cameranavbar">
+      <li className="camerabox">
```

Comment:
It's probably worth having each of these list elements be its own React Component - maybe `RoverCameraButton`?